pkg/network/driver/types.go

```diff
@@ -41,6 +41,7 @@ const FilterPacketHeaderSize = C.sizeof_struct_filterPacketHeader
 type HandleStats C.struct__handle_stats
 type FlowStats C.struct__flow_handle_stats
 type TransportStats C.struct__transport_handle_stats
+type HttpStats C.struct__http_handle_stats
```

Comment:
There's a couple other types missing:
https://github.com/DataDog/datadog-agent/blob/9de8bbf4951a2de0dcf7f7882ae7b3f25dc08598/pkg/network/driver/types.go#L30
https://github.com/DataDog/datadog-agent/blob/9de8bbf4951a2de0dcf7f7882ae7b3f25dc08598/pkg/network/driver/types.go#L77-L85

I suppose we don't really need to any of these, since they aren't going to be used anyhow, but nonetheless I would  prefer if we either add all or none of the types